homestead.root/usr/share/clearwater/cassandra-schemas/homestead_cache.sh

```diff
@@ -1,11 +1,17 @@
 #! /bin/bash
+
+cassandra_hostname="127.0.0.1"
+
 . /etc/clearwater/config
 
 . /usr/share/clearwater/cassandra_schema_utils.sh
 
 quit_if_no_cassandra
 
-if [[ ! -e /var/lib/cassandra/data/homestead_cache ]];
+CQLSH="/usr/share/clearwater/bin/run-in-signaling-namespace cqlsh $cassandra_hostname"
+
+if [[ ! -e /var/lib/cassandra/data/homestead_provisiong ]] || \
```

Comment:
Homestead shouldn't be checking the provisioning database since it doesn't use it. But it *definitely* shouldn't be checking the "provisiong" database ;-)